starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1159,9 +1172,9 @@ sub check_star_catalog {
 	$dither_acq_z = 20.0;
     }
 
-    if (defined $self->{dither_guide}){
-	$dither_guide_y = $self->{dither_guide}->{ampl_y};
-        $dither_guide_z = $self->{dither_guide}->{ampl_p};
+    if (defined $self->{dither_guide_y_max}){
+	$dither_guide_y = $self->{dither_guide_y_max};
+        $dither_guide_z = $self->{dither_guide_z_max}
     } else {
```

Comment:
If I understand the approach now, this case (dither amplitude unknown) should issue a yellow warning since it means something unexpected has happened.  Basically we *always* expect to have valid dither amplitudes for every observation, right?

Reply:
That seems right to me (red or orange would actually seem more appropriate to me at this point).  With regard to kadi dynamic states integration I still have questions (for example, on historical test schedules which may or may not have initial conditions that match kadi?) but should be OK for dither.